Utilities/Reload-Module/Reload-Module.psm1

```diff
@@ -13,9 +13,9 @@
 
 function Reload-Module
 {
-    [System.Diagnostics.CodeAnalysis.SuppressMessageAttribute(
+    [Diagnostics.CodeAnalysis.SuppressMessage(
```

Comment:
I get this when running this from the root script: 
![image](https://user-images.githubusercontent.com/1976647/180021622-a01354bd-61a3-46d5-9a5d-6803c36c63ff.png)

Reply:
Turns out that second empty parameter in SuppressMessageAttribute is necessary after all (but only in PS 5) :(

Reply:
The current code also fails for me (Windows PowerShell):
[output.txt](https://github.com/Lombiq/Utility-Scripts/files/9159378/output.txt)

Reply:
If you run it on the dev branch you will get the same errors too. Also Utility-Scripts are still reloaded. It just fails with the modules in C:\Windows\system32\WindowsPowerShell\v1.0\Modules and C:\Program Files\WindowsPowerShell\Modules. I don't know if we can or should do anything about this, since trying to reload the built-in modules seem pointless anyway.

Reply:
I see, indeed, OK then.

Reply:
I cleaned it up, so there will be no errors.

Reply:
Thank you!